tests/shopify_python/test_google_styleguide.py

```diff
@@ -198,3 +198,30 @@ def fnc():
         message = pylint.testutils.Message('complex-list-comp', node=bad_list_comp)
         with self.assertAddsMessages(message):
             self.walk(root)
+
+    def test_use_cond_exprs(self):
+        # this parsed expression should send a message
+        root_to_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                num = 2
+            """)
+
+        # this parsed expression should NOT send a message
+        root_not_msg = astroid.builder.parse("""
+            if 1 < 2:
+                num = 1
+            else:
+                dum = 2
+            """)
```

Comment:
wouldn't there need to be another value for `other`?
```
if 1<2:
    num, other = 1, 1
else:
    num = 2
```
In this case, you can't really make this a one-liner, as suggested by the google python guidelines, so it should pass the pylint